packages/interface/src/components/SortFilter.tsx

```diff
@@ -8,31 +8,42 @@ import type { OrderBy, SortOrder } from "~/features/filter/types";
 import { SortByDropdown } from "./SortByDropdown";
 import { SearchInput } from "./ui/Form";
 
-export const SortFilter = (): JSX.Element => {
+export interface ISortFilterProps {
+  onSearchChange: (search: string) => void;
+}
+
+export const SortFilter = ({ onSearchChange }: ISortFilterProps): JSX.Element => {
   const { orderBy, sortOrder, setFilter } = useFilter();
 
   const [search, setSearch] = useState("");
   useDebounce(() => setFilter({ search }), 500, [search]);
 
-  const onChange = useCallback(
+  const onChangeSearchInput = useCallback(
     (e: ChangeEvent<HTMLInputElement>) => {
-      setSearch(e.target.value);
+      const term = e.target.value;
+      setSearch(term);
+      onSearchChange(term);
+    },
+    [setSearch, onSearchChange],
+  );
+
+  const onChangeSortByDropdown = useCallback(
+    async (sort: string) => {
+      const [order, sorting] = sort.split("_") as [OrderBy, SortOrder];
+
+      await setFilter({ orderBy: order, sortOrder: sorting }).catch();
     },
-    [setSearch],
+    [setFilter],
   );
```

Comment:
I think the `setFilter` works fine, but changing the sorting is not working for me. 

https://github.com/user-attachments/assets/0763b993-ee3d-4a46-bca6-b4d90d3f504f

Reply:
I focused only on the search bar feature. I created a new issue here https://github.com/privacy-scaling-explorations/maci-platform/issues/600 and I will tackle it soon 😊